website/docs/validating-admission-policy.md

```diff
@@ -177,6 +177,14 @@ spec:
 
 To see this in action, checkout this [demo](https://github.com/open-policy-agent/gatekeeper/tree/master/demo/scoped-enforcement-actions)
 
+Below is the mapping of Gatekeeper's `enforcementActions` to `validatingAdmissionPolicyBinding.spec.validationActions`:
+
+| `enforcementAction` | `validationActions` |
+| --- | --- |
+| `deny` | `Deny` |
+| `warn` | `Warn` |
+| `dryrun` | `Audit` |
```

Comment:
do you want to add this to previous version docs as well but without `dryrun`?

Reply:
Done added till 3.17.

Reply:
did you forget to checkin those files?

Reply:
Yip. My bad 🤦

Reply:
checked in the code. I forgot to push yesterday.